sdk/lib/crt/math/amd64/fabsf.S

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     ReactOS CRT library
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     x64 asm implementation of fabsf
+ * COPYRIGHT:   Copyright 2021 Timo Kreuzer <timo.kreuzer@reactos.org>
+ */
+
+/* INCLUDES ******************************************************************/
+
+#include <asm.inc>
+
+/* CODE **********************************************************************/
+.code64
+
+PUBLIC fabsf
+fabsf:
+    /* Copy parameter into rcx */
+    movq rcx, xmm0
+
+    /* Copy into eax */
+    mov eax, ecx
```

Comment:
Just to know, the higher 32-bits of rax stay equal to zero?

Reply:
They don't stay zero, they will be explicitly set to zero.